index.js

```diff
@@ -3,11 +3,48 @@
 var postcss = require('postcss')
 var isVendorPrefixed = require('is-vendor-prefixed')
 
-module.exports = postcss.plugin('postcss-remove-prefixes', function () {
-  return function removePrefixes (root, result) {
+module.exports = postcss.plugin('postcss-remove-prefixes', function (options) {
+  if (!options) {
+    options = {};
+  }
+
+  var ignore = options.ignore ? Array.isArray(options.ignore) ? options.ignore : false : [];
```

Comment:
With the change above we can nuke this line and change `if (ignore === false)` to `if (!Array.isArray(options.ignore))`.